cns/restserver/restserver.go

```diff
@@ -72,6 +72,7 @@ type HTTPRestService struct {
 	cniConflistGenerator       CNIConflistGenerator
 	generateCNIConflistOnce    sync.Once
 	IPConfigsHandlerMiddleware cns.IPConfigsHandlerMiddleware
+	PnpIDByMacAddress          map[string]string
```

Comment:
What happens to this data on reboot?

Reply:
repopulate

Reply:
you might need to check if `saveState` has been called whenever we update this map, so it can be reconcile after CNS restarted.